.github/workflows/reusable-unit.yml

```diff
@@ -77,9 +78,22 @@ jobs:
         run: yarn ui build
 
       - name: Build react-core package
-        if: ${{ matrix.package == 'react' || matrix.package == 'react-native' || matrix.package == 'react-storage' || matrix.package == 'react-liveness' }}
+        if: |
+          matrix.package == 'react' ||
+          matrix.package == 'react-core-notifications' ||
+          matrix.package == 'react-native' ||
+          matrix.package == 'react-storage' ||
+          matrix.package == 'react-liveness'
         run: yarn react-core build
 
+      - name: Build react-core-notifications package
+        if: |
+          matrix.package == 'react' ||
+          matrix.package == 'react-native' ||
+          matrix.package == 'react-storage' ||
+          matrix.package == 'react-liveness'
```

Comment:
Might be confused here but do we need to build react-core-notifications for react-storage and react-liveness?

Reply:
`react-storage` and `react-liveness` depend on `ui-react`, which depends on `ui-react-core-notifications` because InApp code still lives in `ui-react` package. As we do not have turbo to pull and build each package's dependencies, we are building each package separately in the github workflows.

Reply:
We can stop building `ui-react-core-notifications` for other packages like storage, liveness once we have the breaking change to move InApp code out of `ui-react` package.
 `ui-react` package wouldn't depend on `ui-react-core-notifications` anymore.

Reply:
Let's just make sure we have that captured somewhere as a follow up 😄

Reply:
Added as a subtask in our backlog item for breaking change.